targets/TARGET_STM/TARGET_STM32WL/cmsis.h

```diff
@@ -0,0 +1,24 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2020 STMicroelectronics.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
align license header with us_ticker_data.h for ex

Reply:
Done